### ISSUE TITLE
Automatic Image Populating and Trouble with Exiting Morph Window

### ISSUE THREAD

**Issue by blazetravis** [DCS-LCSR/SignStream3#56]
When retesting issue #10, I encountered another bug.
1. I created a new gloss.
2. I set the new gloss.
3. I left it at one frame to test.
4. I opened up Morph-phon info
5. I tried to change START/END hs and was successfully given a warning message.
6. I went to select handshapes and tried to add 2 handshapes and was successfully given another warning.
7. I deleted the 2 hand shapes and reselected one hand shape.
8. Got the output below in that SignStream populates both the Right and Left hand, even though the gloss was a one-handed sign from the start.

![screen shot 2015-12-01 at 2 02 28 pm](https://cloud.githubusercontent.com/assets/13489587/11511061/dea784c8-9834-11e5-8d3e-bd6d8b8e4a44.png)
1. Went to press enter anyway, and the Morph window would not close.
2. Pressed enter several more times, nothing. So I hit cancel and the window closed, yet it created a new gloss and displayed everything correctly. Strange.

![screen shot 2015-12-01 at 2 08 23 pm](https://cloud.githubusercontent.com/assets/13489587/11511093/0658273e-9835-11e5-8493-5f33972c69ca.png)

**Comment by blazetravis** [comment]
I have a critical update on the last part of this bug (I can make a new bug issue if needed).

Today as I was checking the handshapes for main gloss data, I found the fingerspelled word LOTTERY in Cory 107 U85. The handshapes were all wrong so I opened the morph window and cleared them. Then I selected more appropriate handshapes. I went to enter out of the window to apply the changes, but it would not let me. I knew I had already reported this bug and that sometimes a work-around was to hit cancel to get out of the window and apply changes. However, it seems that this is no longer working because my new handshapes never appeared. I retried several times and cannot get the changes to appear.

This is unfortunately a critical new development that will stifle further gloss correction at this point. I hope that this bug might be considered for the next sprint because of this reason.

Let me know if more information is needed, and thanks for your continued work.
